src/scenekit.cs

```diff
@@ -1550,19 +1722,20 @@ interface SCNScene : NSSecureCoding {
 		[Static, Wrap ("FromFile (name, directory, options == null ? null : options.Dictionary)")]
 		SCNScene FromFile (string name, string directory, SCNSceneLoadingOptions options);
 
-#if MONOMAC
-		[Mac (10,9), iOS (8,0)]
+		// Keeping here the same name WriteToUrl for iOS and friends because it is how it was bound
+		// initialy for macOS and having it named diferently would hurt shared code
+		[TV (10, 0), NoWatch, Mac (10, 9), iOS (10, 0)]
 		[Export ("writeToURL:options:delegate:progressHandler:")]
 		[EditorBrowsable (EditorBrowsableState.Advanced)]
 		bool WriteToUrl (NSUrl url,
 			[NullAllowed] NSDictionary options,
-			[NullAllowed] SCNSceneExportDelegate handler,
+			[NullAllowed] ISCNSceneExportDelegate handler,
```

Comment:
that's good but a breaking change. Did you add some manual code to cover the existing API ?